android/expoview/src/main/java/host/exp/exponent/experience/ReactNativeActivity.java

```diff
@@ -62,7 +63,7 @@
 import static host.exp.exponent.kernel.KernelConstants.LINKING_URI_KEY;
 import static host.exp.exponent.kernel.KernelConstants.MANIFEST_URL_KEY;
 
-public abstract class ReactNativeActivity extends FragmentActivity implements com.facebook.react.modules.core.DefaultHardwareBackBtnHandler {
+public abstract class ReactNativeActivity extends AppCompatActivity implements com.facebook.react.modules.core.DefaultHardwareBackBtnHandler {
```

Comment:
any chance there might be other side effects to this change? 🤔

Reply:
As far as I understand [AppCompatActivity](https://developer.android.com/reference/androidx/appcompat/app/AppCompatActivity) gives you some more features, but without using them explicitly nothing happens 😅